scripts/language_model/data/transform.py

```diff
@@ -0,0 +1,258 @@
+"""XLNet dataset transform."""
+__all__ = ['XLNetSentenceTransform', 'XLNetDatasetTransform']
+import numpy as np
+
+
+
+class XLNetSentenceTransform:
```

Comment:
Can include how this transform differs from `BERTSentenceTransform` in a comment or in the docstring?